bucket/camstudio.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2.7.4-r354",
+    "description": "Free desktop recorder and video streaming software",
+    "homepage": "https://camstudio.org/",
+    "license": "GPL-3.0-or-later",
```

Comment:
"The version available from this website is developed independently under the terms of the GNU Public License V2."
Footer of the website says this, but then in the source code it says v3:
https://sourceforge.net/p/camstudio/mercurial/ci/default/tree/CamStudio2.7/license.txt
What should we put?

Reply:
`gpl-3.0.txt` is also packed in the binaries.

I think they updated the license to GPL-3.0, but didn't update the info on the website.